storage/ndb/tools/NdbImportUtil.cpp

```diff
@@ -1332,7 +1332,7 @@
 NdbImportUtil::Blob::Blob() {
   m_blobsize = 0;
   m_allocsize = 0;
-  m_data = new uchar[0];
+  m_data = nullptr;
```

Comment:
@VarunNagaraju in 8.4.0 I wasn't sure if it is OK to assign `nullptr` to this member here instead of allocating `new char[0]`. So I did `new char[1]`. This `m_data` seems to be a public data member and is accessed from a number of places, so I wasn't sure if changing it to null won't break anything and decided to go `safe` way.